tests/ci/data/profiles/rosa-hcp.yaml

```diff
@@ -11,6 +11,7 @@ profiles:
     byo_vpc: true
     private_link: false
     additional_principals: false
+    imdsv2: "required"
```

Comment:
is `imdsv2` achieved in the provisioning process now?

Reply:
yes